tasks : 
  ButtonA: Slide #O
  ButtonB: Arm #X
  ButtonX: Pick #Δ
  ButtonY: Pass #□
  ButtonLB: Enable
  ButtonRB: Disable
  ButtonSelect: Homing #share
  ButtonTouchpad : Reset
  # ButtonStart: 10 #logo
  # ButtonLeftThumb: 12
  # ButtonRightThumb: 13
  # AxisLeftThumbX: 0
  # AxisLeftThumbY: 1
  # AxisRightThumbX: 2
  # AxisRightThumbY: 3
  # AxisLeftTrigger: 4
  # AxisRightTrigger: 5